app-text/jabref/jabref-2.10-r5.ebuild

```diff
@@ -30,11 +30,11 @@ COMMON_DEP="
 	"
 
 DEPEND="
-	>=virtual/jdk-1.6
+	>=virtual/jdk-1.8:*
```

Comment:
this one does not compile with java 11. it needs `jakarta.xml.bind:jakarta.xml.bind-api:2*`. more info can be seen [here](https://stackoverflow.com/questions/52502189/java-11-package-javax-xml-bind-does-not-exist).

Reply:
> this one does not compile with java 11. it needs `jakarta.xml.bind:jakarta.xml.bind-api:2*`. more info can be seen [here](https://stackoverflow.com/questions/52502189/java-11-package-javax-xml-bind-does-not-exist).

First part coming with https://github.com/gentoo/gentoo/pull/20880